.github/workflows/cd.yaml

```diff
@@ -11,22 +11,7 @@ concurrency:
   cancel-in-progress: true
 jobs:
   build:
-    strategy:
-      matrix:
-        include:
-          - image-name: rdwatch
-            context: "."
-            target:
-          - image-name: builder
-            context: "."
-            target: builder
-          - image-name: postgresql
-            context: docker/services/postgresql
-            target:
-          - image-name: unit-reloader
-            context: docker/services/debug/unit-reloader
-            target:
```

Comment:
Now that we're not using these in development, there's no need to push them as separate images.